stories/dooboo-ui/Card.stories.tsx

```diff
@@ -18,147 +18,102 @@ const Container = styled.View`
-        }}
-        imageStyle={{ width: 200 }}
-        titleContainerStyle={{
-          width: '100%',
-          height: '40px',
-          display: 'flex',
-          flexDirection: 'column',
-        }}
-        title={'This is title'}
-        titleStyle={{ display: 'flex', flexDirection: 'column' }}
-        subTitle={'This is subTitle'}
-        subTitleStyle={{ width: '100%', color: '#e4e4e4' }}>
-        <Text>I have Image</Text>
-      </Card>
-
-      <Card
-        containerStyle={{ marginBottom: 30 }}
-        image={{
-          uri:
-            'https://image.shutterstock.com/z/stock-vector-api-application-programming-interface-software-integration-vector-illustration-1079814893.jpg',
-        }}
-        imageStyle={{ width: 200 }}
-        raised>
-        <Text>{"I'm raised"}</Text>
-      </Card>
-      <Card
-        containerStyle={{ marginBottom: 30 }}
-        image={require('../assets/images/dummy_image_1.jpg')}
-        titleContainerStyle={{
-          width: '100%',
-          height: '40px',
-          display: 'flex',
-          flexDirection: 'column',
-        }}
-        title={'This is title'}
-        titleStyle={{ display: 'flex', flexDirection: 'column' }}
-        subTitle={'This is subTitle'}
-        subTitleStyle={{ width: '100%', color: '#e4e4e4' }}
-      />
-
-      <Card
-        titleContainerStyle={{
-          width: '100%',
-          height: '40px',
-          display: 'flex',
-          flexDirection: 'column',
-        }}
-        title={'This is title'}
-        titleStyle={{ display: 'flex', flexDirection: 'column' }}
-        subTitle={'This is subTitle'}
-        subTitleStyle={{ width: '100%', color: '#e4e4e4' }}>
-        <Text>{"I don't have Image"}</Text>
-      </Card>
-
-      <Card
-        titleContainerStyle={{
-          width: '100%',
-          height: '40px',
-          display: 'flex',
-          flexDirection: 'column',
-        }}
-        title={'This is title'}
-        titleStyle={{ display: 'flex', flexDirection: 'column' }}>
-        <Text>{"I don't have Image"}</Text>
-      </Card>
-
-      <Card
-        titleContainerStyle={{
-          width: '100%',
-          height: '40px',
-          display: 'flex',
-          flexDirection: 'column',
-        }}
-        subTitle={'This is subTitle'}
-        subTitleStyle={{ width: '100%', color: '#e4e4e4' }}>
-        <Text>{"I don't have Image"}</Text>
-      </Card>
-
-      <Card
-        titleContainerStyle={{
-          width: '100%',
-          height: '40px',
-          display: 'flex',
-          flexDirection: 'column',
-        }}>
-        <Text>{"I don't have Image"}</Text>
-      </Card>
-      <Card
-        titleContainerStyle={{
-          width: '100%',
-          height: '40px',
-          display: 'flex',
-          flexDirection: 'column',
-        }}
-        loading={true}>
-        <Text>{"I don't have Image"}</Text>
-      </Card>
-    </Container>
+    <ScrollView style={{ width: '100%' }}>
```

Comment:
Because you use styled-component, it is not recommended to use inline style if possible.
For consistent styling dev env.

Reply:
Inline style excluded.